CMakeLists.txt

```diff
@@ -7,16 +7,16 @@
 # Specify the required version of CMake.
 # cmake 3.10 required for ctest/gtest integration
 cmake_minimum_required(VERSION 3.10)
-project(ale VERSION 0.8.1 DESCRIPTION "Abstraction Library for Ephemerides ")
+project(ale VERSION 0.8.2 DESCRIPTION "Abstraction Library for Ephemerides ")
 
 # include what we need
 include(GNUInstallDirs)
 
 set(CMAKE_CXX_STANDARD 11)
 
 option(ALE_BUILD_LOAD "If the C++ Python load interface should be built." ON)
-option(ALE_USE_EXTERNAL_JSON "If an external nlohmann JSON library should be used" OFF)
-option(ALE_USE_EXTERNAL_EIGEN "If an external EIGEN library should be used" OFF)
+option(ALE_USE_EXTERNAL_JSON "If an external nlohmann JSON library should be used" ON)
+option(ALE_USE_EXTERNAL_EIGEN "If an external EIGEN library should be used" ON)
```

Comment:
These defaulting to OFF causes a weird issue when the ale::ale target gets imported by another CMake project. It doesn't think that nlohmann::json is a dependency and so it never calls find_package.